cmd/backup/config.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	AwsEndpoint                string        `split_words:"true" default:"s3.amazonaws.com"`
 	AwsEndpointProto           string        `split_words:"true" default:"https"`
 	AwsEndpointInsecure        bool          `split_words:"true"`
+	AwsStorageClass            string        `split_words:"true" default:"STANDARD"`
```

Comment:
If I understand the Minio Docs correctly, the default value is the empty string (this is what is used right now anyways), so I think we could get away without the default value here (although the behavior is likely the exact same).